qa/tasks/kafka.py

```diff
@@ -173,6 +173,7 @@ def task(ctx,config):
     tasks:
     - kafka:
         client.0:
+          kafka_version: 2.6.0
```

Comment:
why do you need to specif the kafka version?

Reply:
As per [https://github.com/ceph/ceph/pull/39139#discussion_r594554362](this) we added this parameter which I forgot to add as a reference so just added it here.